packages/cli/test/commands/pull.test.ts

```diff
@@ -46,7 +46,7 @@ describe('pull', () => {
         id: 'vercel-pull-next',
         name: 'vercel-pull-next',
       });
-      client.setArgv('pull', cwd);
+      client.setArgv('pull', cwd, '--yes');
```

Comment:
So technically this is a breaking change - when the env vars are used (cause we won't prompt to link the project). I'm rethinking that we should move this isTTY check to right before the linking prompt happens so that `vc pull` + env vars still doesn't need `--yes`.

Reply:
This test being altered (with the env vars being set) needing to be changed indicates that the error message is happening too soon in the `pull` command. We shouldn't have to specify `--yes` if the env vars have been set since conceptually the project is already "linked" when the env vars are set.